x-pack/package.json

```diff
@@ -129,6 +129,7 @@
     "@samverschueren/stream-to-observable": "^0.3.0",
     "@scant/router": "^0.1.0",
     "@slack/client": "^4.8.0",
+    "@turf/boolean-contains": "6.0.1",
```

Comment:
For anyone else taking a look here, the rest of the deps here are just re-ordered alphabetically (presumably because `yarn add` was used).